wwwroot/src/markup/embed.php

```diff
@@ -50,6 +50,8 @@ function embed_html5_video($input, $options = array())
     }
 
     $attributes = implode(' ', $attributes);
+    if(preg_match("/^https?/", $input))
+        return "<video $attributes><source src='{$input}'></video>";
```

Comment:
I don't like this solution as it could result in video files being embedded from remote hosts. I think if a URL is detected you should use `parse_url` to extract the `path`

Reply:
"Video files from remote hosts" is a feature not a bug; we want to use CDNs to host the video files, not just serve the video files from the sync instance  ie. clients start their session with theatre.wetfish.net, but video files are served from a geolocated loadbalancing cache cdn.wetfish.net .

Reply:
@mozai this is for the wiki. This has nothing to do with the sync project